R/JASPDescriptivesPlot.R

```diff
@@ -0,0 +1,105 @@
+
+#' Descriptive plot of categorical variable (x-axis) vs. a continuous variable (y-axis)
+#'
+#' @param x character vector indicating the groups shown on the x-axis.
+#' @param y numeric vector indicating the y-values.
+#' @param ciLower numeric vector with the lower bound of the confidence interval.
+#' @param ciUpper numeric vector with the upper bound of the confidence interval.
+#' @param xName character, name to use as the x-axis title.
+#' @param yName character, name to use as the y-axis title.
+#' @param group character vector indicating group membership, which is used for shape and fill color
+#' @param groupName character, name to show as the legend title.
+#' @param horizontalAsymptote numeric, draw a horizontal asymptote at this y-value.
+#' @param horizontalAsymptoteLineType line type of the horizontal asymptote.
+#' @param position a value for the position argument of e.g., geom_line.
+#' @param lineSize a value for the size argument of geom_line.
+#' @param pointSize a value for the size argument of geom_point.
+#' @param errorbarWidth a value for the width argument of geom_errorbar.
+#' @param hideXLevelNames Logical, should the level names on the x-axis be hidden?
+#' @param showBreaksAtExtremaOnly Should only the outermost y-axis breaks be shown?
+#' @param doNotConnectPoints Should the dots be connected with lines?
```

Comment:
Can we formulate it in the active sense? I always find double negatives (`doNotDoSomething = FALSE`) confusing.